src/controllers/subject.js

```
@@ -23,11 +23,8 @@ const getSubjectByname = async (req, res) => {
 }
 
 const createSubject = async (req, res) => {
-  const { id: categoryId, category: categoryName } = req.user
   const data = req.body
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)